backend/queries/time.js

```diff
@@ -54,10 +45,19 @@ const deleteHoursByVolunteerId = async (id, promise) => {
   return await db.any(deleteQuery, id);
 }
 
+const updateVolunteerHours = async (v_id, newHours) => {
```

Comment:
This function is not used anymore, we are tracking volunteered hours in the event_volunteers table